app/model/editions/EditionsCard.scala

```diff
@@ -28,13 +28,24 @@ object CoverCardImages {
   implicit val format: OFormat[CoverCardImages] = Json.format[CoverCardImages]
 }
 
-
-case class Palette(paletteId: String, foregroundHex: String, backgroundHex: String)
+case class Palette(foregroundHex: String, backgroundHex: String)
```

Comment:
ID is now a property of the meta theme, not the palette, because a theme can contain multiple palettes.